gpii/node_modules/processReporter/configs/gpii.processReporter.config.development.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
@kaspermarkus Irony:  there's a similar discussion on [GPII-1839](https://issues.gpii.net/browse/GPII-1839) ("Always use the dynamic device reporter") about using mock device reporters for integration testing.  There, [I noted](https://issues.gpii.net/browse/GPII-1839?focusedCommentId=22421&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-22421):

> The difference between the static DR and the mock dynamic DR is slight. The static DR simply grabs a list of "installed solutions" and echoes it back. The mock DR looks at the list and simulates the platform specific function for searching it ...

Mostly rhetorical:  if the matchmaker is confused by a PR that always reports the same state, why isn't it confused by a DR that always reports that everything is installed?

It looks like a kind of convergence to me:  always use the dynamic device reporter and the dynamic process reporter, even in development mode.

No problem in making this change; I'm on it.